### ISSUE TITLE
Show required fields lables on add item page by default 

### ISSUE THREAD

**Issue by KkageG** [enBloc-org/kindly#190]
# Expected Behaviour

```
Required fields lables are shown on add item page by default
```

# Current Behaviour

```
Required fields lables are shown on add item page only after picking an image to upload
```

## Steps to Reproduce
```
1. Run the application locally 
2. Log in as a user 
3. Go to add item page http://localhost:3000/add-item 
4. Pick an image to upload (a valid jpeg file) 
5. Click  "add your item" button 
```
<img width="1680" alt="Screenshot 2024-04-29 at 00 29 00" src="https://github.com/enBloc-org/kindly/assets/149285326/dbf5090e-a48f-4de7-ac03-631a08a43952">

**Comment by nataliiazab** [comment]
I would like to solve this issue

**Comment by camelPhonso** [comment]
@nataliiazab I've assigned you to the issue now - it's worth reaching out to @AlexVOiceover over some changes previously made into a branch related to this issue

Thanks for the help ✨

**Comment by nataliiazab** [comment]
Hi @AlexVOiceover, would you like to pair program to solve this issue or to quickly update me on your changes? 😃

**Comment by AlexVOiceover** [comment]
Hi Natalie,

 

I put this on hold because the full time program forced me to finish some other tasks. I didn’t work in an iterative way because it was mostly research, so I need to reorganise the code and I will update the code this Sunday.

The problem here is that the required fields check is being done in two times. The component that manages the image load has it’s own check, the problem is that it will hold the check of the form until his is resolved, therefore you need to resolve the image before the rest of the fields are checked.

I fixed it by moving the image check to the form and not the component itself using a simple flag to keep track if the image was loaded.

I also encountered many other mistakes like missing IDs and similar, but I will roll back those to keep the issue simple

 

Alexander Rodríguez

 <https://www.alexvoiceover.com/> AlexVOiceover.com

07908314467

This message is for the sole use of the person (s) to whom it is addressed and may contain confidential information.

Any unauthorized review, use, disclosure or distribution is prohibited.

If you are not the intended recipient, please contact the sender and delete all copies from your system, your cooperation is appreciated.

**Comment by Angelica137** [comment]
!request

Can I work on this issue?

**Comment by AlexVOiceover** [comment]
> !request
> 
> Can I work on this issue?

Yes please, feel free to take over

**Comment by AnyaVeselova** [comment]
!request

I'd like to work on the issue if no one is working on it.
